onnx/defs/controlflow/old.cc

```diff
@@ -8,6 +8,792 @@
 namespace ONNX_NAMESPACE {
 using SupportType = OpSchema::SupportType;
 
+static std::vector<std::string> control_flow_types_ir4() {
+  auto t = OpSchema::all_tensor_types_ir4();
+  auto s = OpSchema::all_tensor_sequence_types_ir4();
+  auto o = OpSchema::all_optional_types_ir4();
+  t.insert(t.end(), s.begin(), s.end());
+  t.insert(t.end(), o.begin(), o.end());
+  return t;
+}
+
+void IfInferenceFunction_16(InferenceContext& ctx) {
```

Comment:
Is there any change to the inference function? If not, it would be nice to avoid code-duplication and use a common function. The other folders use a `utils.h` and `utils.cc` for code shared by `defs.cc` and `old.cc`